style.css

```diff
@@ -1,134 +1,430 @@
-            
-            .infoList {
-                list-style-type: none;
-                padding-right: 100px;
-                margin-right: 30px;
-                margin-top: 50px;
-            }
-            
-            .pokeImg {
-                float: left;
-                height: 320px;
-                position: relative;
-            }
-            
-            .divInfoList {
-                float: right;
-                height: auto;
-                width: auto;
-                font-size: 30px;
-                text-align: right;
-            }
-            
-            .infoName {
-                height: 230px;
-                width: auto;
-                ;
-            }
-            
-            .pokeNameParagraph {
-                margin-left: 35px;
-                padding-left: 60px;
-                padding-top: 60px;
-                font-size: 44px;
-                float: right;
-                margin-right: 130px;
-            }
-            
-            .displayDateAndTimeParagraph {
-                top: 520px;
-                left: 820px;
-                position: fixed;
-                width: 190px;
-                height: 40px;
-                color: #7bd573;
-            }
\ No newline at end of file
+                @use postcss-color-function;
+                @use postcss-nested;
+                @import url('https://fonts.googleapis.com/css2?family=Bangers&display=swap');
+                @import url('https://fonts.googleapis.com/css2?family=Grandstander:wght@100&display=swap');
+                body {
+                    height: 80vh;
+                    width: 100vw;
+                    font-weight: 100;
+                }
+                
+                .shadow {
+                    z-index: -1;
+                    position: fixed;
+                    top: 475px;
+                    width: 1050px;
+                    height: 100px;
+                    border-bottom-right-radius: 50px;
+                    border-bottom-left-radius: 50px;
+                    background-color: #600606;
+                }
+                
+                .list {
+                    height: 550px;
+                    width: 300px;
+                    background-color: #c10c0c;
+                    float: left;
+                    border-top-left-radius: 65px;
+                    border-bottom-left-radius: 45px;
+                    box-shadow: -14px 12px 0px 5px #600606;
+                }
+                
+                .listOfPokemons {
+                    height: 390px;
+                    overflow: scroll;
+                    scrollbar-width: none;
+                    -ms-overflow-style: none;
+                }
+                
+                .listElement {
+                    margin: 0;
+                    padding: 15px 30px;
+                    list-style: none;
+                    background-image: url(https://img.icons8.com/color/48/000000/pikachu-pokemon.png);
```

Comment:
Please download file and use it locally instead of urls

Reply:
Fixed!